src/main/java/seedu/address/ui/cards/CompanyCard.java

```diff
@@ -1,27 +1,75 @@
 package seedu.address.ui.cards;
 
-import seedu.address.model.person.Person;
+import static seedu.address.ui.panel.PanelDisplayKeyword.ADDRESS_DISPLAY_NAME;
+import static seedu.address.ui.panel.PanelDisplayKeyword.COMPANY_DISPLAY_NAME;
+import static seedu.address.ui.panel.PanelDisplayKeyword.EMAIL_DISPLAY_NAME;
+import static seedu.address.ui.panel.PanelDisplayKeyword.INDUSTRIES_DISPLAY_NAME;
+import static seedu.address.ui.panel.PanelDisplayKeyword.PHONE_DISPLAY_NAME;
+
+import seedu.address.model.company.CompanyItem;
 
 /**
  * todo Javadocs
  */
-public class CompanyCard extends Card {
+public class CompanyCard extends Card<CompanyItem> {
 
     /**
      * todo Javadocs
      */
-    public CompanyCard(Person person, int displayedIndex) {
-        super(person, displayedIndex);
+    public CompanyCard(CompanyItem companyItem, int displayedIndex) {
+        super(companyItem, displayedIndex);
         initializeCompanyCardGui();
     }
 
     /**
      * todo Javadocs
      */
     private void initializeCompanyCardGui() {
-        // to be edited in the future!
-        initializeHeader(displayedIndex);
-        initializeBody();
-        initializeTags();
+        setId(displayedIndex);
+        setName();
+        setTags();
```

Comment:
Same thing regarding the tags naming here. Unless I'm misunderstanding the use of the word "tag" in the context of this UI portion of code. Apologies if that's the case!

Reply:
yup I think I should change this to `setIndustries`